index.html

```diff
@@ -319,7 +319,9 @@ <h3><dfn>DefaultGraph</dfn> interface</h3>
     <h3><dfn>Quad</dfn> interface</h3>
 
     <pre class="idl">
-    interface Quad {
+    interface Quad : Term {
+      attribute string? termType;
+      attribute string? value;
```

Comment:
But what would the value be?

Reply:
I would not make these fields optional.

`termType` can easily be defined as `Quad` and it is not a breaking change to an implementation which does not support RDF*. They will simply ignore it

`value`, I'm not sure. Just make it a constant `undefined` or empty string?

Reply:
I would say making these fields required _is_ a breaking change. Existing datafactories will be _required_ to expose them, and existing consumers of datafactories can break if old datafactories are used.

Having a `value` field here with a constant string or `undefined` feels too much like a hack to me, so I would not be in favour of that.

Reply:
I think you're looking at the breaking change backwards.

Yes, a factory which upgrades to RDF* version will need to adjust (either adding new fields to `Quad` or adding `DataFactory#quadTerm` method.

Those are not important however. What is important is whether a library which depends on the current model spec would break once it is enhanced to support RDF*.

The answer is no, AFAICT.

Reply:
Good point, I was looking at it backwards.

Only the `value` issue remains then.

Reply:
Ha, and I was just about to reply similarly, having read your [other comment](https://github.com/rdfjs/data-model-spec/pull/163#issuecomment-656543500)

Trying to use "old quads" in RDF* factory indeed is not going to be seamless 🤔